Sources/SwiftProtobuf/JSONEncodingError.swift

```diff
@@ -12,12 +12,12 @@
 ///
 // -----------------------------------------------------------------------------
 
-public enum JSONEncodingError: Error {
+public enum JSONEncodingError: Error, Equatable {
```

Comment:
Same

Reply:
Without it, I get the following error when doing `swift test`:

```
/Users/michael/Development/swift-protobuf/Tests/SwiftProtobufTests/Test_JSON_Conformance.swift:298:7: error: global function 'XCTAssertEqual(_:_:_:file:line:)' requires that 'JSONEncodingError' conform to 'Equatable'
      XCTAssertEqual($0 as? JSONEncodingError,
      ^
Swift.Optional:1:11: note: requirement from conditional conformance of 'Optional<JSONEncodingError>' to 'Equatable'
extension Optional : Equatable where Wrapped : Equatable {
          ^
```

Reply:
Thanks. I figured it was going to be implicit equatable support, but wanted to confirm.